src/popups/PopupContextMenu.as

```diff
@@ -8,10 +8,14 @@ package popups
     import flash.events.MouseEvent;
     import flash.filters.BlurFilter;
     import flash.geom.Point;
+    import flash.profiler.showRedrawRegions;
+
     import menu.MenuPanel;
 
     public class PopupContextMenu extends MenuPanel
     {
+        private static var redrawBoolean:Boolean = false;
```

Comment:
```suggestion
        CONFIG::debug
        {
            private static var redrawBoolean:Boolean = false;
        }
```
Since this variable is only used for showing redraw regions in debug mode, I think we could place this in `CONFIG::debug`.